spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/mapping/SpannerCompositeKeyProperty.java

```diff
@@ -112,6 +111,7 @@ public String getName() {
 	}
 
 	@Override
+	@Deprecated
```

Comment:
Just confirming we haven't added this by accident.

Reply:
The method this implements on the interface is annotated as deprecated. Should we also annotate it like that (in case the user is using an instance of this directly rather than an instance of the interface)?